Improve "Why?" and "What?" Section of README.md #9

nelsonic opened the pull request:
On re-re-re-reading the doc, I felt it could be improved. Ref #9 

@iteles please take a look and give feedback when you can. 👍 
If you're happy with these changes, please merge. :shipit: 
Thanks. 🙏 

> **Note**: Sorry about the mega update. future changes will be far more incremental. 📝

nelsonic committed:
improve "Why?" section #9

nelsonic committed:
add clarity on the *system* we are trying to build

nelsonic commented:
Every time I end up writing too much text I think: this would be fine as a "Blog Post" or an "essay",
but it's _way_ too much to force someone to read for a Product Roadmap ... 💭

nelsonic committed:
remove duplicate words to reduce repetition and speed up reading

nelsonic committed:
update 7 critical questions in Why? section

nelsonic committed:
remove excess <br /> from list

nelsonic committed:
add intro note on "work-in-progress" nature of Product Roadmap

nelsonic committed:
tidy up copy in Why? section

nelsonic commented on README.md:
```diff
@@ -69,6 +276,37 @@ that together help the individual
 
+
+
+## 1. _Individual_ Person Experience -> [_Single Player_](https://en.wikipedia.org/wiki/Single-player_video_game)
```
I'm busy expanding on the **_Individual_ Person's Experience** as the _first_ thing we are building.
related to: https://github.com/dwyl/app/issues/213 (Create Personas)

nelsonic replied:
We can add personas in the next PR.

nelsonic committed:
add link to product UX research repo: https://github.com/dwyl/product-ux-research

nelsonic committed:
add final note to end of doc

nelsonic committed:
tidy up line length of individual person experience section

nelsonic committed:
add link to https://github.com/dwyl/app/issues/213 Personas issue to prioritise it!

nelsonic committed:
increasingly thinking this needs to be split out into separate pages ... 💭

nelsonic committed:
Notion encourages busy-work and I’m tired of it

nelsonic committed:
add note + quote on prioritising

nelsonic committed:
re-org intro paragraph

nelsonic commented:
Continue: https://github.com/dwyl/product-roadmap/tree/update-what-section-issue%239

nelsonic committed:
Finish sentences and re-org "What?" section #9

nelsonic committed:
Merge branch 'master' into update-what-section-issue#9

nelsonic commented:
@iteles I've fixed the Merge Conflicts on this branch/PR so it's ready for review. ♻️ 
Please take a look when you have some time. 🙏 
Thanks very much. 🌻

nelsonic commented:
@SimonLab please have a read through this and let us know if it makes sense. 🙏

SimonLab commented on README.md:
```diff
@@ -1,38 +1,130 @@
+## _Why_ Are We Building An _App_?
+
+If there was an _existing_ 
+(Open Source Cross-platform Real-time Collabroative) 
```
```suggestion
(Open Source Cross-platform Real-time Collaborative) 
```

SimonLab commented on README.md:
```diff
@@ -1,38 +1,130 @@
+"Reminder", 
+"Calendar" (_Meeting, Date, Event, Travel, etc._), 
+"Ingredient" (Shopping list `item`), etc. 
+We want _complete_ versaility in the `tags` that can be appplied to items
```
```suggestion
We want _complete_ versatility in the `tags` that can be applied to items
```

SimonLab commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+## "_I don't have enough time to_..."
+
+How many times have you heard 
+a friend, family member or work colleage 
```
```suggestion
a friend, family member or work colleague 
```

SimonLab commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+and **don't** actually **_help_** us **move _forward_**!
+In the case of the one mentioning Beyonce, it's an outright _lie_.
+**Queen Bey** has an _army_ of people helping her get the most out of her day.
+The rest of us don't have personal assitants and entourages of people
```
```suggestion
The rest of us don't have personal assistants and entourages of people
```

SimonLab commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+TODO: Create Persona for this person: https://github.com/dwyl/app/issues/213
+-->
+
+The _reason_ most people are _innefective_ at focussing
```
```suggestion
The _reason_ most people are _ineffective_ at focussing
```

SimonLab commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+
+
+The ***system*** we are building
+is desgined to help ourselves and others 
```
```suggestion
is designed to help ourselves and others 
```

SimonLab commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+  
+  > Capturing text (_and later voice memos and images_) as "items" 
+  which can then be categorised/organised using tags 
+  is the most intiuitive and versatile way of storing data.
```
```suggestion
  is the most intuitive and versatile way of storing data.
```

SimonLab commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+  We only need _one_ field or data type (`text`).
+  We can then apply `tags` to an `item` to make it into _anything_ else.
+  And then the UI renders the item with the appropriate UX 
+  for an `item` with the `tag` `"task"`, display a checkbox next to it.
```
```suggestion
  for an `item` with the `tag` `"task"`, displaying a checkbox next to it.
```

nelsonic replied:
This is technically acceptable grammar; the action is to "_display_" the checkbox.

SimonLab commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+  to have goals defined for each area of a person's life. 
+  We will have an on-boarding questionnaire to guide them, 
+  a visualisation of "progress" toward each one of the goals 
+  and reminder to review them regularly. 
```
```suggestion
  and reminders to review them regularly. 
```

SimonLab commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+### 2. Small _Team_ Experience -> [_Co-op Mode_](https://en.wikipedia.org/wiki/Cooperative_gameplay)
+ 
+
+We feel that our App will be _incredibly_ 
```
```suggestion
We feel that our App will be _incredibly_ helpful
```

SimonLab commented on README.md:
```diff
@@ -101,8 +537,10 @@ The @dwyl App Will have the following basic workflow:
 to allow people to
-input **`plaintext`**
-and save it to a backend as **`text`**.
+input **`palintext`**
```
```suggestion
input **`plaintext`**
```

SimonLab commented on README.md:
```diff
@@ -101,8 +537,10 @@ The @dwyl App Will have the following basic workflow:
-and save it to a backend as **`text`**.
+input **`palintext`**
+and save it to a backend as **`text`**
+in one ore more **`items`**.
```
```suggestion
in one or more **`items`**.
```

SimonLab commented on README.md:
```diff
@@ -387,14 +846,43 @@ and be _frugal_ with bandwidth.
+
+## _Public_?
+
+From a "commercial" perspective making our
```
```suggestion
From a "commercial" perspective, making our
```

SimonLab commented on README.md:
```diff
@@ -428,3 +916,57 @@ but only because the knowledge is **not** shared openly_.
+No map is a _complete_ representation of the world;
+it's just a "model" that allows us to think about it.
+When you open Google Maps on your phone
+it does not show you all the street lights and advertising bilboards.
```
```suggestion
it does not show you all the street lights and advertising billboards.
```

SimonLab commented on README.md:
```diff
@@ -428,3 +916,57 @@ but only because the knowledge is **not** shared openly_.
+Instead, that part of our long-term vision
+is summarised as "API". 
+Having a REST+Realtime API will allow
+anone to inteact with their data 
```
```suggestion
anyone to interact with their data 
```

SimonLab reviewed (commented):
I found a few typos:

nelsonic committed:
Fix typo Collabroative > Collaborative

Glad to see someone is reading close enough to catch these. Collaborative indeed. 😉

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix typos: versaility > versatility & appplied > applied

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix typo: assitants > assistants

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix typo: colleage > colleague

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix typo: innefective > ineffective

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix typo: desgined > designed

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix typo: palintext > plaintext ... though I quite like Palin Text https://en.wikipedia.org/wiki/Michael_Palin

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix typo: ore > or

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix grammar by adding a comma to break sentence.

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix typos: anone > anyone & inteact > interact

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix typo: bilboards > billboards

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix typo: intiuitive > intuitive

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix grammar: display > displaying

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix grammar: reminder > reminders (pluralize)

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Complete the sentence by adding the word "helpful".

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic commented:
@SimonLab thanks for making all these grammar & typo corrections. ✅ 
If you have any other feedback, please continue to comment. 🎉

nelsonic commented:
@iteles, Simon has kindly done a first-pass of reading this PR and made several typo fixes. 🙌 
Please take a look when you can and give feedback. Thanks. 👍

SimonLab commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+please open an issue!
+
+
+## 3. Big Company/Org Experience -> [_MMORPG_](https://en.wikipedia.org/wiki/Massively_multiplayer_online_role-playing_game)!
```
Third point of the stages of the roadmap should have the same type of title to the two previous points
```suggestion
### 3. Big Company/Org Experience -> [_MMORPG_](https://en.wikipedia.org/wiki/Massively_multiplayer_online_role-playing_game)!
```

nelsonic committed:
Fix heading level of section 3. Thanks @SimonLab

Co-authored-by: Simon <simon.labondance@gmail.com>

SimonLab committed:
add missing "on", https://github.com/dwyl/product-roadmap/pull/22#issuecomment-924164641

SimonLab committed:
Merge branch 'update-what-section-issue#9' of github.com:dwyl/product-roadmap into update-what-section-issue#9

SimonLab commented on README.md:
```diff
@@ -142,15 +580,24 @@ And finally for the _third_ step, ***Complete***,
-The **answer** is: _initially_ it may _appear_ that our App
+The **answer** is: 
+_initially_ it may _appear_ that our App
 does not have much different
```
```suggestion
does not differ much
```

SimonLab commented on README.md:
```diff
@@ -1,38 +1,130 @@
+<br />
 
-## _Why_?
+# _Why_? 🤷‍♀️
```
because the hierarchy of the headings (from "##" to "#") has changed, we also need to update the other sections (Who, When, How)

nelsonic replied:
@SimonLab agreed. If you have time, please update. 📝

SimonLab replied:
It should be fixed with https://github.com/dwyl/product-roadmap/pull/22/commits/0862462ef4d7c39f04bead23ecb6a55d08a19c1a

nelsonic replied:
Cool. I will ignore the notifications till you're done. 👍

SimonLab replied:
:+1: I think I'm almost done with formatting. I've been reading the document and I'll add comments/issue focused on the content itself.

nelsonic replied:
Defo ask questions and even open issues for things that are unclear. 👍

nelsonic replied:
Great exploratory question: https://github.com/dwyl/product-roadmap/issues/27 Thanks @SimonLab 🎉

SimonLab commented on README.md:
```diff
@@ -1,38 +1,130 @@
+We want _complete_ versatility in the `tags` that can be applied to items
+so that our App can be used for _any_ purpose that helps people organise their lives.
+
+5. ***Track*** tasks and goal **progress _visually_**.
```
Keep same Emphasis as previous points
```suggestion
5. **Track** tasks and goal progress **_visually_**.
```

nelsonic replied:
Same again. the "Track" word is an _action_ that's why it's italicised. Happy to remove and revisit. 👍

SimonLab commented on README.md:
```diff
@@ -1,38 +1,130 @@
+
+5. ***Track*** tasks and goal **progress _visually_**.
+
+6. ***Communicate*** with others what task we are focussing on
```
```suggestion
6. **Communicate** with others what task we are focussing on
```

nelsonic replied:
Same here: "Communicate" is an _action_ word. So italics emphasise it. But I _agree_ that I use italics _way_ too often ... 🙄

SimonLab commented on README.md:
```diff
@@ -1,38 +1,130 @@
+Avoiding the "what you doing" type questions,
+or general interruptions is essential for focus work.
+
+7. ***Delegate*** specific tasks 
```
```suggestion
7. **Delegate** specific tasks 
```

nelsonic replied:
Ah, the reason this italicised is that it's an _action_. We need a style guide. 💭 
Meanwhile applying your suggestion for consistency. 👍

SimonLab committed:
update headings, https://github.com/dwyl/product-roadmap/pull/22/files#r713810159

SimonLab reviewed (commented):
Update emphasis:

nelsonic committed:
remove italics from "Delegate" word.

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix grammar: "does not have much different" > "does not differ much". Thanks @SimonLab

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Fix markdown of "Track" sentence.

Co-authored-by: Simon <simon.labondance@gmail.com>

nelsonic committed:
Remove italics from "Communicate" word.

Co-authored-by: Simon <simon.labondance@gmail.com>

iteles commented on README.md:
```diff
@@ -1,38 +1,130 @@
+We want to apply both the science and art of distraction-free focus
+we have read about in books such as 
+"Deep Work",
+"Indestractable", 
```
```suggestion
"Indistractable", 
```

nelsonic replied:
Good catch. Thanks for fixing. 👍 
Ref: https://www.amazon.com/Indistractable-Control-Your-Attention-Choose/dp/194883653X

iteles commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+Few people know more about effectiveness 
+than life coach and motivational speaker 
+[Tony Robbins](https://en.wikipedia.org/wiki/Tony_Robbins). <br />
+A quote of his that guides us in our mission is:
```
Tony Robbins has become somewhat controversial because there are dozens of sexual harassment and abuse of power claims against him so I would be tempted to tone this down a bit.

```suggestion
Although Tony Robbins is a somewhat polarising character, we unreservedly agree with this thought:
```

iteles replied:
Tony Robbins has become somewhat controversial because there are dozens of sexual harassment and abuse of power claims against him so I would be tempted to tone this down a bit as suggested above just so it's not distracting.

nelsonic replied:
Yeah, this makes me very sad. 😢 
+ https://www.buzzfeednews.com/article/katiejmbaker/tony-robbins-accused-sexual-assault-teenager-supercamp
+ https://graziadaily.co.uk/life/opinion/tony-robbins-sexual-assault-and-harassment-allegations/

We can just remove the section or sub it for an alternative... 💭

iteles commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+## _Empower people to maximise effectiveness, creativity and happiness._
+
+
+Few people know more about effectiveness 
```
```suggestion
```

iteles commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+
+
+Few people know more about effectiveness 
+than life coach and motivational speaker 
```
```suggestion
```

iteles commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+
+Few people know more about effectiveness 
+than life coach and motivational speaker 
+[Tony Robbins](https://en.wikipedia.org/wiki/Tony_Robbins). <br />
```
```suggestion
```

iteles commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+about our time-management - _**self**_ 
+"**productivity** [***shaming***](https://en.wikipedia.org/wiki/Shame)" -
+and **don't** actually **_help_** us **move _forward_**!
+In the case of the one mentioning Beyonce, it's an outright _lie_.
```
```suggestion
```

iteles replied:
I don't think this is necessary given the following sentence.

nelsonic commented:
We all get side-tracked ...

![image](https://user-images.githubusercontent.com/194400/148379736-0c7bee29-0fc7-4929-8cca-36f9029c2c29.png)

![image](https://user-images.githubusercontent.com/194400/148379776-8b004b2f-d6b0-4ac0-a190-f24c36dcea03.png)

Staying focussed on-task is a _surprisingly_ hard task for a lot of people. ⏳ 
If we can help them _solve_ it systematically, it will be a _huge_ hit. 💭

iteles commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+is designed to help ourselves and others 
+move towards our
+(life/health/fitness/career/etc.) 
+goals is what we are building.
```
```suggestion
goals.
```

iteles replied:
Repeated from the beginning of the sentence

iteles commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+(life/health/fitness/career/etc.) 
+goals is what we are building.
+We haven't found a catchy one-line phrase
+to summarise the ***system*** we are building. ([`#HelpWanted`]())
```
```suggestion
to summarise it yet. ([`#HelpWanted`]())
```

iteles commented on README.md:
```diff
@@ -59,20 +151,364 @@ of these challenges.
+
+There is no reason why a "Big" company or organisation cannot use the dwyl App.
+But this is not our focus right now. 
+We need to crawl, walk & run before we attempt to _fly_. 
```
👍

nelsonic replied:
👍

iteles reviewed (dismissed):
My apologies for taking so long to review this. I started reviewing way back last year and was scattered after that.
Thank you as ever for advancing this @nelsonic and adding to the team motivation in moving this forward!

iteles commented:
@nelsonic Could you please take a quick look and approve/decline the suggested changes and then I'll do the final merge?

nelsonic committed:
Update spelling of Indistractable README.md

Co-authored-by: Ines Teles Correia <iteles@users.noreply.github.com>

nelsonic committed:
remove Beyonce line.

Co-authored-by: Ines Teles Correia <iteles@users.noreply.github.com>

nelsonic committed:
tighten sentence on goals.

Co-authored-by: Ines Teles Correia <iteles@users.noreply.github.com>

nelsonic committed:
remove excess use of "system".

Co-authored-by: Ines Teles Correia <iteles@users.noreply.github.com>

nelsonic committed:
Clarify mention of Tony Robbins https://github.com/dwyl/product-roadmap/pull/22#discussion_r719337910

Co-authored-by: Ines Teles Correia <iteles@users.noreply.github.com>

nelsonic committed:
Remove superfluous sentence

Co-authored-by: Ines Teles Correia <iteles@users.noreply.github.com>

nelsonic committed:
cut "life coach" line

Co-authored-by: Ines Teles Correia <iteles@users.noreply.github.com>

nelsonic committed:
Remove link to Tony Robbins (Wikipedia)

Co-authored-by: Ines Teles Correia <iteles@users.noreply.github.com>

nelsonic commented:
@iteles all your suggestions are applied. Thanks! 🎉

nelsonic commented:
Thank you for reviewing/improving & merging this @iteles ❤️
